brut.j.util/src/main/java/brut/util/OS.java

```diff
@@ -72,11 +72,11 @@ public static void cpdir(File src, File dest) throws BrutException {
                 continue;
             }
             try {
-                InputStream in = new FileInputStream(file);
-                OutputStream out = new FileOutputStream(destFile);
-                IOUtils.copy(in, out);
-                in.close();
-                out.close();
+                try(InputStream in = new FileInputStream(file)){
+                    try(OutputStream out = new FileOutputStream(destFile)){
+                        IOUtils.copy(in, out);               
+                    }
+                }
```

Comment:
```
try (InputStream in = new FileInputStream(file)) {
    try (OutputStream out = new FileOutputStream(destFile)) {
        IOUtils.copy(in, out);               
    }
}
```

Suggested spacing fixes

Reply:
Sorry, I don't have reformat the code.
I must reformat and resubmit a pull request?

Reply:
Just another commit is fine, no need for another PR

Reply:
I have update the PR with a new commit